test/integration/rewrites-has-condition/test/index.test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import {
+  findPort,
+  killApp,
+  launchApp,
+  nextBuild,
+  nextStart,
+} from 'next-test-utils'
+import webdriver from 'next-webdriver'
+
+jest.setTimeout(1000 * 60 * 2)
+
+const appDir = join(__dirname, '../')
+
+let appPort
+let app
+
+const runTests = () => {
+  it('should load page rewrite without browser errors', async () => {
+    const browser = await webdriver(appPort, '/rewrite-simple')
+
+    expect(await browser.waitForElementByCss('#another').text()).toBe(
+      'another page'
+    )
+
+    const browserLogs = await browser.log('browser')
+    const errorLogs = browserLogs.filter((log) => {
+      return log.level.name === 'SEVERE' && log.message.includes('Error:')
```

Comment:
This may make the test flaky if other errors are to be expected in the future.

The alternative is to test explicitly whether `fromEntries` is in the error message.

```suggestion
      return log.level.name === 'SEVERE' && log.message.includes('fromEntries')
```